client/app/index.html

```diff
@@ -50,7 +50,14 @@
                         </div>
 
                         <!-- /Skip content target -->
-                        <ovh-chatbot></ovh-chatbot>
+                        <ovh-chatbot
+                            data-can-switch-to-livechat="true"
+                            data-config="{
+                                universe: 'WEB',
+                                language: $ctrl.currentLanguage,
+                                country: $ctrl.user.ovhSubsidiary
```

Comment:
won't you have an issue is user is not loaded yet and you display the chatbot? do you have some kind of loader to avoid this issue?

Reply:
I thought about it. For that to happen, you'll need to have a user with a slow connection who by-passes the main loader and decides to click on the Chatbot button while everything else is loading. I considered that an edge case. There's currently no way to know if the chatbot is available or not from the navbar, because they are decoupled components.